### PR TITLE
KIALI-1033 Fix duplicates in filtering in ServiceList

### PR DESCRIPTION
**Pull request by aljesusg**
Jira: [KIALI-1033](https://issues.jboss.org/browse/KIALI-1033)

filtering

### CODE REVIEW

**Comment by lucasponce** [comment]
Thanks @aljesusg. Do you mind to review if the same issue happens in the Istio Config ? As it is using a similar filtering ? Perhaps it could apply the same solution there.

**Comment by aljesusg** [comment]
Mmm I am gonna check it @lucasponce

**Comment by aljesusg** [comment]
@lucasponce @jotak I fixed the istioconfig list too

## Before
before

## After
after


We have the same result with filters @jotak Should we open a JIRA for this?

File: src/utils/__tests__/Common.test.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { removeDuplicatesArray } from '../Common';
+
+const arrayDuplicates = ['bookinfo', 'deafault', 'bookinfo'];
```
Comment (lucasponce) [inline src/utils/__tests__/Common.test.ts]: typo: 'deafault' -> 'default'

Comment (aljesusg) [reply src/utils/__tests__/Common.test.ts]: arghh ^^

**Comment by aljesusg** [comment]
Thanks @lucasponce

**Pull request merged**